### ISSUE TITLE
Questions about parameter settings during model training

### ISSUE THREAD

**Issue by zhangxinzxr** [cviviers/YOLOv5-6D-Pose#27]
I used my own dataset for training, the image size is 1920*1080, epochs are set to 200, and the output result is as below when the label truth value and for detection, what are the possible reasons?

![Image](https://github.com/user-attachments/assets/b7e59341-2822-45e1-8775-c4724fc99bb2)

![Image](https://github.com/user-attachments/assets/0a029560-c019-4174-bbca-e167c0835503)

**Comment by cviviers** [comment]
Hi @zhangxinzxr,

Interesting application. Can you render the label with the YOLO code in the [dataloader](https://github.com/cviviers/YOLOv5-6D-Pose/blob/6b982c6043bb41ab53da804e8d09300ed5204261/utils/datasets.py#L745), just to make sure the labels are loader correctly. If this is correct then it could be something with the anchor box threshold. You can increase the size [here](https://github.com/cviviers/YOLOv5-6D-Pose/blob/6b982c6043bb41ab53da804e8d09300ed5204261/configs/hyp.single.yaml#L19).

Do you see the losses go down during training?

**Comment by zhangxinzxr** [comment]
Thank you for your answer, here is the loss of the training process. And notice that the image in media in wandb is also strange.The output information during training represents a very large error.
-----------------------------------
  tensor to cuda : 0.001976
         predict : 1.568805
    compute loss : 0.011172
get_region_boxes : 0.002495
            eval : 0.127207
             pnp : 0.000200
           total : 1.584447
-----------------------------------
   Mean corner error is 1031.174194
   Acc using 5 px 2D Projection = 0.00%
   Acc using 0.045750000000000006 vx 3D Transformation = 0.00%
   Acc using 5 cm 5 degree metric = 0.00%
   Translation error: 3024.268153, angle error: 68.800206
(1031.1742, 0.0, 0.0, 0.0, 0.3599945306777954, 0.0582573339343071, 0.0, 60.12156677246094)
Saving best model with fitness: [          0]

![Image](https://github.com/user-attachments/assets/a92da7fc-55ae-484c-8cc9-5f7e559d69b6)
![Image](https://github.com/user-attachments/assets/d884acc6-033b-406e-8a8e-c9418acddeb4)

**Comment by cviviers** [comment]
Alright, it seems to be doing the right thing now - just requires more training. You can train for longer at the current LR or increase the default LR if you dont want to train for more epochs. Looking at the loss curves you provided, it is clear that the network is still busy converging.

**Comment by lainqsw** [comment]
朋友你好，请问你是从哪里下载到rov6D的数据集以及使用的模型的？我看好像没有完整开源

**Comment by zhangxinzxr** [comment]
> 朋友你好，请问你是从哪里下载到rov6D的数据集以及使用的模型的？我看好像没有完整开源

你好，rov6d数据集确实是只开源了部分水池环境下的数据。可以在bluerov的官网下载sw模型，另存为step文件，使用在线的转换工具转换成二进制的ply文件后，简单的程序即可转换成ASCII格式的ply文件。但是我们只需要模型外形的ply文件，内部结构尽量隐藏，不然文件会很大，我也没找到啥很有效的方法。
